### PR TITLE
added Text Editor like Notepad in java

### PR DESCRIPTION
**Pull request by SyedAhmed334**

### CODE REVIEW

File: src/main/java/org/example/Java Text Editor/TextEditor.java [hunk 1]
```diff
@@ -0,0 +1,428 @@
+import java.awt.*;
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/Java Text Editor/TextEditor.java:1]: package info is missing

File: src/main/java/org/example/Java Text Editor/MyFileStack.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
+public class MyFileStack {
```
Comment (BimalRajGyawali) [inline src/main/java/org/example/Java Text Editor/MyFileStack.java:1]: package info is missing

**Review by BimalRajGyawali — CHANGES_REQUESTED** [review]

**Comment by SyedAhmed334** [comment]
It is basically java swing, This package will automatically be available if
u have jdk. And ofcourse if you don't have jdk u won't be able to run java.

**Comment by BimalRajGyawali** [comment]
By package info I meant 
```java 
package org.example.somepackage;
```

Also follow naming conventions as per https://github.com/BimalRajGyawali/hacktoberfest-java/blob/main/CONTRIBUTING.md#naming-conventions

**Pull request closed without merging**